core/src/main/java/io/squashql/query/dto/PivotTableQueryDto.java

```diff
@@ -1,19 +1,19 @@
 package io.squashql.query.dto;
 
+import io.squashql.query.Field;
+import java.util.List;
 import lombok.AllArgsConstructor;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import lombok.ToString;
 
-import java.util.List;
-
 @ToString
 @EqualsAndHashCode
 @NoArgsConstructor // For Jackson
 @AllArgsConstructor
 public class PivotTableQueryDto {
 
   public QueryDto query;
-  public List<String> rows;
-  public List<String> columns;
+  public List<Field> rows; //todo-mde should we use alias since we know rows and columns are already in the query ?
```

Comment:
It is ok I think. Users will use the same variables as the ones defined for the query.